src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.cs

```diff
@@ -25,27 +25,52 @@ internal sealed class ODataUtf8JsonWriter : IJsonWriter, IDisposable
         private readonly float bufferFlushThreshold;
 
         private readonly Stream outputStream;
+        private readonly Stream writeStream;
         private readonly Utf8JsonWriter writer;
         private readonly int bufferSize;
         private readonly bool isIeee754Compatible;
         private readonly bool leaveStreamOpen;
-        private readonly Encoding encoding = Encoding.UTF8;
+        private readonly Encoding utf8Encoding = Encoding.UTF8;

```

Comment:
I'm not seeing any way this will be something other than `Encoding.UTF8`. Can't we just refer to it directly (in fact, we do in the constructor)?

Reply:
Yeah, you're right about that. Will make the change.